packages/babyjubjub/src/field.ts

```diff
@@ -73,6 +73,10 @@ export class Field extends BN {
     return prefix.or(this)
   }
 
+  toJSON(): string {
+    return `0x${super.toJSON()}`
```

Comment:
I'm a little worried about breaking `bn.js`'s original toJSON function. Do you think there will be no side-effect of this? And this function has a duplicated functionality with `toHex()` function.

Reply:
`bn.js` returns a hex string without the `0x` prefix by default. When creating a `new BN()` if there is a `0x` in the string it will throw an error (in version of `bn.js` >= 5).

There's some discussion [here](https://github.com/indutny/bn.js/issues/151).

I think if we bump the `bn.js` version we should be fine, it will fail loudly instead of silently returning an incorrect bn instance.

Reply:
Actually the project already uses `bn.js@^5.1.1` so we should be good. Going to mark this as resolved.